python/ml4ir/applications/classification/config/parse_args.py

```diff
@@ -0,0 +1,30 @@
+from argparse import Namespace
+
+from ml4ir.base.config.keys import TFRecordTypeKey
+
+from ml4ir.applications.ranking.config.keys import LossKey, MetricKey
+from ml4ir.base.config.parse_args import RelevanceArgParser
+
+from typing import List
+
+
+class ClassificationArgParser(RelevanceArgParser):
+    """
+    States default arguments for classification model.
+    """
+    def set_default_args(self):
+        super().set_default_args()
+        self.set_defaults(
+            tfrecord_type=TFRecordTypeKey.EXAMPLE,
+            loss_key=LossKey.CATEGORICAL_CROSS_ENTROPY,
+            metrics_keys="['" + MetricKey.CATEGORICAL_ACCURACY + "'"
```

Comment:
@Ullimague @balikasg Why are you not using MRR and ACR for EP?

Reply:
At the beginning you mentioned MRR and ACR wouldn't work for non-ranking (as it needs a rank info in the dataset). Currently we can live without it.

Reply:
Okay. Yes, there is a bug with the shapes and need for `rank` to use MRR and ACR. I'm asking because we can prioritize fixing it if you need these metrics. Looks like it's not urgent. Sounds good.